doc/app/sm_build_and_run.rst

```diff
@@ -256,7 +210,7 @@ See :ref:`sm_testing` for typical test cases.
 .. _sm_carrier_library_support:
 
 Using the LwM2M carrier library
-===============================
+*******************************
```

Comment:
I think this could be moved to a separate subpage of its own?

Reply:
At some point yes. Right now I've just kept everything from "Building and running" here.